content/2019 -1-8_trinacty-sraz-co-dal.md

```diff
@@ -0,0 +1,39 @@
+Title: Třináctý sraz - Co dál?
+Date: 2019-1-8 18:00:00
+Modified: 2019-1-8 18:00:00
+Author: Anežka Müller
+
+Od posledního srazu uplynula poměrně dlouhá doba, neviděly jsme se mezi svátky ani první lednový týden, protože setkání připadalo na 1.1.
+Rekapitulace, co kdo dělal, ale nebyla nijak obsáhlá, vzhledem k tomu, že pauza zahrnovala hlavně vánoční svátky a Silvestra. Každá z nás pracovala na nějaké separátní části kódu, které se věnovala už předtím. Také je nás opět o něco méně, aktuální počet členek týmu je 5. 
+
+**Co dál?**
+
+Hlavní otázka setkání byla, do čeho se po svátcích pustit. Začly jsme tedy probírat postupně jednotlivé issues a aktuální pull requesty našeho projektu. Tím jsme si i shrnuly, v jakém je projekt stavu. Třeba u dokumentačních řetězců byla na začátku situace taková, že je nikdo nepsal, po založení issue došlo k jejich doplnění a nyní si na to již většinou dáváme pozor a píšeme dokumentační řetězce zároveň s kódem. 
+Ačkoliv to na začátku vypadalo, že nevíme, co dál, z issues vyplynulo docela dost úkolů. Nemáme dokončené testy pro validaci mapy, což je poměrně komplexní úkol, ale nebrzdí nás v dalším vývoji. Velkým tématem je rozhraní hry. Důležité zde je, stejně jako u herního plánu, aby byl stav oddělen od toho, co se poté vykresluje do okénka. 
+
+![stav_rozhrani](./images/stav_rozhrani.jpg)
```

Comment:
nezobrazuje se mi v preview obrázek, nemá třeba moc velké rozlišení nebo něco?

Reply:
Nevím, mně to funguje normálně a je to stejný formát i velikost, jako moje fotky v předchozích příspěvcích.

Reply:
> nezobrazuje se mi v preview obrázek, nemá třeba moc velké rozlišení nebo něco?

Nevím, mně to funguje normálně a je to stejný formát i velikost, jako moje fotky v předchozích příspěvcích (má 3MB)

Reply:
já právě když jsem dělala můj blog s obrázkama tak jsem je musela zmenšovat protože se mi nevykreslily